docs/tools/getlit-cli.md

```diff
@@ -33,4 +35,105 @@ To use the GetLit CLI, navigate to a directory or your existing project, and the
 | `watch`           | `getlit watch [<lit-action-name>]`  | 🔧 Simultaneously build and test a Lit Action |
 | `setup`           | `getlit setup`                      | 🔑 Setup config for authSig and PKP      |
 | `docs` \| `doc` | `getlit docs`                       | 📖 Open the Lit Protocol documentation   |
-| `help` \|  `show` | `getlit help`    | 🆘 Show the help menu                     |
\ No newline at end of file
+| `help` \|  `show` | `getlit help`    | 🆘 Show the help menu                     |
+
+### Basic Application
+
+```
+getlit action
+```
+
+Initialized Lit project directory looks like:
+
+```
+├── README.md
+├── getlit.json
+├── globa.d.ts
+├── package-lock.json
+├── package.json
+├── src
+│   ├── README.md
+│   ├── foo.action.ts
+│   └── main.action.ts
+├── test
+│   ├── README.md
+│   ├── foo.t.action.mjs
+│   └── main.t.action.mjs
+├── tsconfig.json
+└── utils.mjs
+```
+
+In order to proceed, `src/foo.action.ts` needs to be modified as ‘NA_E’ to ‘NAME’:
+
+```javascript
+/**
+ * NA_E: foo
+ *
+ * ⬆️ Replace "_" with "M" to pass the schema validation
+ *
+ */
+ 
+const foo = () => {
+  return "bar";
+};
+```
+
+Let's see what we have in our main Lit Action, `src/main.action.ts`:
+
+```javascript
+/**
+ * NAME: hello
+ */
+ 
+// This will exceed the default file size limit
+// import * as LitJsSdk from "@lit-protocol/lit-node-client-nodejs";
+ 
+type SignData = number[];
+ 
+const helloWorld: SignData = [
+  72, 101, 108, 108, 111, 32, 87, 111, 114, 108, 100,
+];
+ 
+(async () => {
+  // this requests a signature share from the Lit Node
+  // the signature share will be automatically returned in the HTTP response from the node
+  const sigShare = await LitActions.signEcdsa({
+    toSign: new Uint8Array(helloWorld),
+    publicKey, // <-- You should pass this in jsParam
```

Comment:
perhaps make this some constant like `<your pkp publickey>` so users know they have to fill it in.